CONTRIBUTING.md

```diff
@@ -100,6 +101,50 @@ Be sure to include any related GitHub issue references in the commit message.  S
 [GFM syntax](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) for referencing issues
 and commits.
 
+
+[dronevic]:https://ci.vmware.run/vmware/vic
+[e2edronevic]:https://e2e.ci.vmware.run/vmware/vic
+[dronesrc]:https://github.com/drone/drone
+[dronecli]:http://readme.drone.io/devs/cli/
+
+## Automated Testing
+
+Automated testing uses [Drone][dronesrc].
+
+PRs must pass unit tests and integration tests before being merged into `master`.
+
+You can run the tests locally before making a PR or view the Drone build results for [unit tests][dronevic]
+and [integration tests][e2edronevic].
+
+If you don't have a running ESX required for tests, you can leverage the automated Drone servers for
+running tests. Add `WIP` (work in progress) to the PR title to alert reviewers that the PR is not ready to be merged.
+
+
+### Testing locally
+
+Developers need to install [Drone CLI][dronecli].
+
+#### Unit tests
+
+``` shell
+drone exec --yaml .drone.yml -e VIC_ESX_TEST_URL="<USER>:<PASS>@<ESX IP>"
+```
+
+If you don't have a running ESX, tests requiring an ESX can be skipped with the following:
+
+``` shell
+drone exec --yaml .drone.yml -e VIC_ESX_TEST_URL=""
+```
+
+#### Integration tests
+
+Integration tests require a running ESX on which to deploy VIC.
+
+``` shell
+drone exec --yaml .drone-e2e.yml -e VIC_ESX_TEST_URL="<USER>:<PASS>@<ESX IP>"
```

Comment:
This file (.drone-e2e.yml) looks missing from this changeset.  Or am I just not seeing it?